src/community/china.md

```diff
@@ -66,10 +66,19 @@ If you’re interested in setting up your own mirror in China,
 contact [flutter-dev@googlegroups.com](mailto:flutter-dev@googlegroups.com)
 for assistance.
 
-If you're running into issues that only occur when using the `flutter-io.cn` server,
-consider reporting the issue to
+If you're running into issues that only occur when
+using the `flutter-io.cn` server, consider reporting the issue to
 [the issue tracker (镜像问题)](https://github.com/cfug/flutter.cn/issues/new/choose).
 
+## Publishing apps
+
+If you receive errors when attempting to publish your package,
```

Comment:
I don't think it's fully covered/fixed the issue. The most worth mention part is: reset the `PUB_HOSTED_URL` during publishes. So it's better to write: run `unset PUB_HOSTED_URL` in the publishing.

Reply:
And in fact, these kind of happened when the host server does not supports Google OAuth, it's not just China specific.

Reply:
So I can see there are some issues for now and maybe in the future, related to China. Please let me acknowledge them so I can help to produce a proper solution. :)

Reply:
@AlexV525 : Thanks for the update! I'll await your instructions.

Reply:
Thanks! Following this should cover the issue https://github.com/flutter/website/pull/9338#discussion_r1328077020